### ISSUE TITLE
Are there no videos?

### ISSUE THREAD

**Issue by Mandrewoid** [pyconca/2017-web#77]
Hi, I was planning on going to Pyconca 2017 but unfortunately I got sick and was unable to go. Are there no videos from this year?

**Comment by frdeso** [comment]
Yes, we recorded the talks, we're currently doing the editing and it's taking more time than expected. They will be uploaded on YouTube when ready.
